application/classes/Ushahidi/Core.php

```diff
@@ -73,6 +73,11 @@ public static function init()
 				and Kohana::$config->load('features.private.enabled');
 		});
 
+		// Intercom config settings
+		$di->set('site.intercomAppToken', function() use ($di) {
+			return Kohana::$config->load('site.intercomAppToken');
```

Comment:
hmmmm, I hadn't thought of that. I added it to site to allow users to edit it for their deployment not with the intention that they can push updates to it. Would it be better under something like third_party_app_config?